content/departments/product-engineering/product/design/workshops/async-design.md

```diff
@@ -0,0 +1,61 @@
+# Async design workshop
+
+If you want to avoid sync meetings and want to have a design workshop activity that can involve a large group of people you can consider an async design workshop.
+
+The activities you do on a given day are flexible with this approach, so you can do problem validation, solution validation, brainstorming, and more; the important part is making it a multi-day activity with offsetting days for content creation and evaluation/voting, with a date boundary in between.
+
+Benefits:
+
+- Supports async work
+- More flexibility when people are in lots of time zones
+- Can support quite large groups
+- Self-documenting
+- Allows time for reflection and "shower thoughts" to be included
+
+Drawbacks:
+
+- Plays out slower than if everyone was in a live session together
+- Not as engaging for people who are inspired by real-time communication
+- Requires lots of written communication
+
+## Scheduling considerations
+
+Because this is an activity that takes a little bit of attention each day over a few days or a week, its' good to avoid scheduling at the same time as the release is wrapping up, or another heavyweight organizational activity like planning is going on.
+
+## Offsetting days
+
+The first thing you should do when planning the workshop is identify the list of activities you want to complete. Then, order them across multiple days with one day doing the activity, and the next day summarizing and voting. This rhythm of having a day for producing information and a day for summarizing it/voting on it keeps the team making forward progress, and supports people participating in all time zones. It also is highly tolerant of people joining late or dropping out early.
+
+See [this example of a complete agenda](#example-agenda) for how you might structure something like this in detail.
+
+## Deliverables
+
+On the non-offset days, it's important to structure your deliverables so that they can be shared easily. Written tends to be ideal, but videos are also good. So, if you're planning on doing some problem validation, you might have everyone record a short video of problems they are aware of with the product today.
```

Comment:
```suggestion
On the non-offset days, it's important to structure your deliverables so that they can be shared easily. Simple is best - letting people share off the cuff thoughts in the Slack channel has the lowest barrier to entry, and will encourage the most participation. Beyond that, written tends to be ideal, but videos are also good. So, if you're planning on doing some problem validation, you might have everyone record a short video of problems they are aware of with the product today.
```